enterprise/backend/src/metabase_enterprise/workspaces/api.clj

```diff
@@ -248,7 +256,15 @@
   (when database_id
     (check-transforms-enabled! database_id))
 
-  (ws->response (ws.common/create-workspace! api/*current-user-id* body)))
+  ;; If no database_id provided, use first supported DB as provisional default (uninitialized workspace)
+  (let [provisional (not database_id)
+        database-id (or database_id
+                        (api/check-400 (first-supported-database-id)
```

Comment:
nit: I think this should be a 5xx, since there's nothing wrong with the request itself?

Not a blocker for me.

Reply:
Well you as a FE came to create workspaces before it is possible, so it's 400. I'd rather use 500 for unexpected stuff only. :)